### PR TITLE
Added script for removing marc:nonfilingChars when the value equals ' '

### PR DESCRIPTION
**Pull request by lrosenstrom**

### CODE REVIEW

**Comment by lrosenstrom** [comment]
Test run against STG took ~20min.

File: whelktool/scripts/cleanups/2019/09/remove-marc:nonfilingChars-with-empty-string-value/remove-marc:nonFilingChars.groovy [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ */
+
+PrintWriter failedIDs = getReportWriter("failed-to-delete-Ids")
+PrintWriter scheduledForChange = getReportWriter("scheduled-for-change")
+
+where = "data#>>'{@graph,1,hasTitle,0,marc:nonfilingChars}' = ' '"
+def absentNonfilingChars = ["marc:nonfilingChars": " "]
+
+selectBySqlWhere(where, silent: false) { docItem ->
+    Map hasTitle = docItem.doc.data["@graph"][1]["hasTitle"][0]
```
Comment (niklasl) [inline whelktool/scripts/cleanups/2019/09/remove-marc:nonfilingChars-with-empty-string-value/remove-marc:nonFilingChars.groovy]: I think there may be nonfilingChars in any Title item, not just the first?

File: whelktool/scripts/cleanups/2019/09/remove-marc:nonfilingChars-with-empty-string-value/remove-marc:nonFilingChars.groovy [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    hasTitle.remove(absentNonfilingChars)
```
Comment (niklasl) [inline whelktool/scripts/cleanups/2019/09/remove-marc:nonfilingChars-with-empty-string-value/remove-marc:nonFilingChars.groovy]: Does this really drop the key?

File: whelktool/scripts/cleanups/2019/09/remove-marc:nonfilingChars-with-empty-string-value/remove-marc:nonFilingChars.groovy (hunk 1 above)
Comment (lrosenstrom) [reply whelktool/scripts/cleanups/2019/09/remove-marc:nonfilingChars-with-empty-string-value/remove-marc:nonFilingChars.groovy]: Yes, me and @sarwik discussed this. I'm reworking it.

Comment (niklasl) [reply whelktool/scripts/cleanups/2019/09/remove-marc:nonfilingChars-with-empty-string-value/remove-marc:nonFilingChars.groovy]: Ah, perfect; now I understand the "close without merge"! :)

**Pull request closed without merging**